### DISCUSSION TITLE
[Script request]  SimpleX Chat Server

### DISCUSSION THREAD

**Discussion by Node815** [community-scripts/ProxmoxVE#722] [Request script]
### Application Name

SimpleX 

### Website

https://simplex.chat/docs/server.html#installation

### Description

SimpleX is one of a few encrypted chat systems.   It's a zero knowledge based chat and allows you to host your own server if you should so desire, or use their server.    Since the chats are never decrypted by the server, it remains zero knowledge. They also assign a random user ID to the chatters so it remains completely anonymous.  You can choose to keep your ID if you wish so it never changes, or take advantage of the truly random one they provide each session.   

They offer Android and iPhone clients which allow the end user to chat through the network.  Having our own server would help satisfy even the most privacy conscious individuals and their families.  

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by sero2kk** [comment]
I would also like to see a script like this :-))
